qiskit/visualization/matplotlib.py

```diff
@@ -785,8 +785,8 @@ def _draw_ops(self, verbose=False):
                 elif len(q_xy) == 1:
                     disp = op.name
                     if param:
-                        prm = '({})'.format(param)
-                        if len(prm) < 20:
+                        prm = '{}'.format(param)
```

Comment:
This line can be more obvious on what's doing!
```suggestion
                        prm = str(param)
```